### ISSUE TITLE
Performance issue: Single String concatenation 

### ISSUE THREAD

**Issue by mmalaquiasdev** [Yorubaname/yorubaname-website#67]
Hi,

In this case when we have a single concatenation:

```
response.put("message", new StringBuilder(name).append(" not found in the repository so not indexed").toString());
```
It's more efficient and simpler using a simple String concatenation.

```
response.put("message", name + " not found in the repository so not indexed");
```

**Comment by mmalaquiasdev** [comment]
solved #64

**Comment by mmalaquiasdev** [comment]
Merged #69
